### PR TITLE
Build result view columns from query metadata

### PR DESCRIPTION
**Pull request by ThePrez**

### CODE REVIEW

File: src/views/results/index.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ class ResultSetPanelProvider {
           this._view.webview.postMessage({
             command: `rows`,
             rows: queryResults.data,
+            columnList: queryResults.metadata.columns.map(x=>x.name),
```
Comment (worksofliam) [inline src/views/results/index.ts:53]: This is likely an issue because `metadata` is not (and should not) be returned on every fetch.

**Review by worksofliam — CHANGES_REQUESTED** [review]
@ThePrez Little issue. Works perfect for the first fetch when the meta data from the server is returned, but when trying to fetch the next page it fails with this error:

image

This is with a simple `select * from qsys2.columns`.

File: src/views/results/index.ts (hunk 1 above)
Comment (worksofliam) [reply src/views/results/index.ts:53]: edit: looks like you solved this exact issue in PR #93

Comment (ThePrez) [reply src/views/results/index.ts]: Plucked to https://github.com/halcyon-tech/vscode-db2i/pull/90/commits/c5178ef0a0b719f6dbc7280a2386adadeb1ce21f

**Review by worksofliam — APPROVED** [review]

**Pull request merged**